h2o-docs/src/product/mojo-quickstart.rst

```diff
@@ -693,6 +693,54 @@ If you cannot install Graphviz on your environment, another option is to produce
   :alt: Example Output Picture
   :scale: 50%
 
+GBM MOJO to ONNX Converter
```

Comment:
this is not accurate, the json output is just a intermediate format which is then used by ONNXMLTools to generate the final ONNX model.

https://github.com/onnx/onnxmltools#h2o-to-onnx-conversion

Reply:
@hannah-tillman just change the heading to "Printing GBM MOJOs"